src/components/BottomSheet/components/BottomSheetBackdrop.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {memo, useMemo} from 'react'
+import {TapGestureHandler, TapGestureHandlerGestureEvent} from 'react-native-gesture-handler'
+import Animated, {
+  runOnJS,
+  useAnimatedGestureHandler,
+  useAnimatedStyle,
+  withTiming,
+} from 'react-native-reanimated'
+import styled from 'styled-components'
+import {useBottomSheet} from '../hooks/useBottomSheet'
```

Comment:
After adding useBottomSheet to index.ts, please shorten this import

Reply:
updated